EpsilonAuxBmsV2/Tests/tests/OrionInterfaceTest/main.c

```diff
@@ -5,8 +5,18 @@
 #include "stm32f4xx_hal_can.h"
 #include "OrionInterfaceTest.h"
 
+extern OrionCanInfo message;
+// extern AuxStatus auxStatus;
+// extern AuxTrip auxTrip;
+// extern AuxBmsContactorState auxBmsContactorState;
+// extern osEventFlagsId_t contactorControlEventBits;
```

Comment:
Just curious, where are these defined now that they've been commented out?

Reply:
Woops, I forgot to commit the latest changes.